clients/test/plugins/plugin-test/src/test/response/string.ts

```diff
@@ -15,9 +15,10 @@
  */
 import { TestStringResponse } from '@kui-shell/test'
 
-/* eslint-disable no-new */
-new TestStringResponse({
+const test = new TestStringResponse({
   command: 'test string',
   expect: 'hello world',
   exact: true
 })
+
+test.string()
```

Comment:
Modified the string response test file to avoid using `new` without storing the result. See Eslint Rule no-new: https://eslint.org/docs/1.0.0/rules/no-new